src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -50,6 +53,12 @@ const PermissionIdentities: FC = () => {
   const [searchParams] = useSearchParams();
   const [selectedIdentityIds, setSelectedIdentityIds] = useState<string[]>([]);
   const { hasAccessManagementTLS } = useSupportedFeatures();
+  const isSmallScreen = useSmallScreen();
+  const { openPortal, closePortal, isOpen, Portal } = usePortal();
+
+  const handleClose = () => {
```

Comment:
no need to have this function here, we can just pass `closePortal` to `onClose` below